bolt_store.go

```
@@ -97,7 +98,11 @@ func (b *BoltStore) initialize() error {
 	if err != nil {
 		return err
 	}
-	defer tx.Rollback()
+	defer func() {
+		if err := tx.Rollback(); err != nil {
+			log.Printf("Rollback failed: %v", err)
```

**Issue:** Instead of logging the error and failing silently, let's just return the `error` back like it's done at line 99. 
Please follow same approach for all the places.

---

Seems like `defer tx.Rollback` will anyways ignore the error silently, so to avoid a breaking change, let's stick to your current implementation of logging the error only.